src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -627,7 +627,7 @@ else if (participation.getExercise() instanceof ProgrammingExercise) {
     @PutMapping("/participations/{id}/cleanupBuildPlan")
     @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
     public ResponseEntity<Participation> cleanupBuildPlan(@PathVariable Long id, Principal principal) {
-        ProgrammingExerciseStudentParticipation participation = (ProgrammingExerciseStudentParticipation) participationService.findOneStudentParticipation(participationId);
+        ProgrammingExerciseStudentParticipation participation = (ProgrammingExerciseStudentParticipation) participationService.findOneStudentParticipation(id);
```

Comment:
we should not use `id`, but should always write in the name what kind of id we have, in this case a `participationId`

Reply:
I fixed it, also for some other places in this file